apps/frontend/src/components/Login.tsx

```diff
@@ -0,0 +1,79 @@
+import React, { useState } from "react";
+import styles from "../assets/login.module.css";
+import { useNavigate } from "react-router-dom";
+
+export default function Login() {
+  const navigate = useNavigate();
+
+  // 🔹 States pour les champs de formulaire
+  const [username, setUsername] = useState("");
+  const [password, setPassword] = useState("");
+  const [error, setError] = useState("");
+
+  // 🔹 Fonction pour envoyer la requête à ton backend
+  const handleLogin = async () => {
+    try {
+      const response = await fetch("http://localhost:3000/api/login", {
```

Comment:
the backend url should use an env variable instead of a string to localhost. Check .env.example for the list of env vars you need to have on your local machine 👍 
```suggestion
      const response = await fetch(`${import.meta.VITE_BACKEND_URL}/api/login`, {
```

Reply:
I will change it with you because when I try It raise an error that Vite.. is not reconize